crates/enarx-config/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Do we need a separate LICENSE in the subcrate or can we reuse the one from the root of the workspace?
@bstrie you mentioned we'd need explicit licenses in https://github.com/enarx/enarx/pull/2020#discussion_r913161527, does this mean we also need to have LICENSE files in each subcrate?